public/posts/chapter-117.md

```diff
@@ -3,6 +3,77 @@ title: 'Safe Internet Practices'
 date: '2024-09-28'
```

Comment:
Lets keep today's date. 😉

Reply:
Besides that, its good to go!!! 
Thanks for your contribution, @aditi-manthri

Reply:
> Lets keep today's date. 😉

Sure, let me just change that!